content/05-mysql/12-short-guides/01-creating-deleting-users.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'How to use create and delete users in MySQL'
```

Comment:
I'd maybe change the title (and meta title) here again to make "How to use create..." sound a bit more fluid.